app/controllers/application_controller.rb

```diff
@@ -15,7 +15,10 @@ class ApplicationController < ActionController::Base
   authorize :user_with_groups
 
   def user_with_groups
-    UserWithGroups.new(user: current_user, groups: ldap_groups) if current_user
+    UserWithGroups.new(user: current_user, groups: ldap_groups)
+  rescue Devise::MissingWarden
+    # A nil UserWithGroups, indicating that not logged in
```

Comment:
This should never happen.   If we are seeing this. Then components will not be showing up on the page (after a live update) when they should

Reply:
Can this change be reverted?

Reply:
What? I thought I did. Looking.

Reply:
Apparently it only works if you hit save in VSCode ...